src/System.Net.Requests/tests/AuthenticationManagerTest.cs

```diff
@@ -12,29 +12,41 @@ namespace System.Net.Tests
 {
     public class AuthenticationManagerTest
     {
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "AuthenticationManager supported on NETFX")]
```

Comment:
The goal right now is not to port all tests from .NET Framework.  The goal is to get the existing tests in CoreFx to work reliably.